docs/book/v3/features/error-handling.md

```diff
@@ -213,6 +221,16 @@ class LoggingErrorListenerDelegatorFactory
 }
 ```
 
+Then, enable the delegator factory in your application, such as by adding the following to the `getDependencies()` function in your app or module’s _ConfigProvider.php_ file.
+
+```php
+'delegators' => [
+    ErrorHandler::class => [
+        LoggingErrorListenerDelegatorFactory::class,
```

Comment:
I'll be honest, the naming is really terrible (although pre-existing) 😬

Reply:
True. I was focused on sticking as close to the existing documentation in this PR as possible. Happy to come back with a larger PR that reworks the naming used throughout this section of the docs.

Reply:
Closing after accepting @froschdesign's update.